inference-engine/ie_bridges/c/samples/.clang-format

```diff
@@ -0,0 +1,25 @@
+BasedOnStyle: Google
```

Comment:
I suppose these files will be installed as a part of these commands https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/CMakeLists.txt#L75-L112

can we add the to exclude list?

Secondly, should not we have the common clang-format config for the whole IE? CC @ilyachur are going to align IE and ngraph code styles?

Reply:
We shouldn't add the global clang-format config, because in this case clang-format will be automatically applied for all changed files if developer integrated clang-format to text editor.